cores/esp32/esp32-hal-bt.c

```diff
@@ -17,6 +17,9 @@
 #if SOC_BT_SUPPORTED
 #ifdef CONFIG_BT_BLUEDROID_ENABLED
 
+#if __has_include("esp_bt.h")
+#include "esp_bt.h"
+
```

Comment:
I think that you should update the upper if, rather than adding a new one. One reason is to provide default implementations for the functions in this file.
```cpp
#if defined(CONFIG_BT_BLUEDROID_ENABLED) && __has_include("esp_bt.h")
```